tests/foreman/cli/test_host.py

```diff
@@ -2683,3 +2683,71 @@ def test_positive_create_host_with_lifecycle_environment_name(
     hosts = module_target_sat.cli.Host.list({'organization-id': module_org.id})
     found_host = any(new_host.name in i.values() for i in hosts)
     assert found_host, 'Assertion failed: host not found'
+
+
+@pytest.mark.no_containers
+@pytest.mark.rhel_ver_match('8')
+@pytest.mark.parametrize(
+    'setting_update', ['validate_host_lce_content_source_coherence'], indirect=False
+)
+def test_positive_host_registration_with_capsule(
+    target_sat,
+    setting_update,
+    function_sca_manifest_org,
+    function_activation_key,
+    rhel_contenthost,
+    capsule_configured,
+):
+    """Register client with capsule when settings "validate_host_lce_content_source_coherence" is set to Yes/No
+
+    :id: 17dbec62-eed4-4a51-9927-80919457a124
+
+    :BZ: SAT-22048
+
+    :setup:
+        1. Create AK which is associated with the organization
+        2. Configure capsule with satellite
+    :steps:
+        1. Register client with capsule when settings "validate_host_lce_content_source_coherence" is set to Yes
+        2. Check output for "HTTP error code 422: Validation failed: Content view environment content facets is invalid"
+        3. Re-register client with settings "validate_host_lce_content_source_coherence" is set to No
+        4. Check output there should not any error like "Validation failed" or "HTTP error code 422"
+
+    :expectedresults: Host registration success with error when settings set to Yes and Host registration success when
+     settings set to No
+
+    :customerscenario: true
+
+    :CaseImportance: Medium
+    """
+    # set a new proxy
```

Comment:
you could also use this way the property name, this will be helpful in case you have more than 1 properties 

`property_name = setting_update.name`

Reply:
I could see refernce of the above session is mostly use in UI tests, I will use the same way which has impletemented. Let me know if anything else we could use here.